src/components/Personalization/index.js

```diff
@@ -40,26 +41,66 @@ const createCollect = collect => {
     });
 };
 
-const createStore = () => {
+const createStorage = () => {
   const storage = [];
 
-  return (selector, meta) => {
-    storage.push({ selector, meta });
+  return {
+    add: ({ selector, meta }) => {
+      storage.push({ selector, meta });
+    },
+    getAll: () => {
+      return storage;
+    }
   };
 };
 
+// Using node selection vs matches selector, because of :eq()
+const matches = (selector, node) => {
+  const nodes = selectNodesWithEq(selector, node);
+
+  return nodes.length > 0;
+};
+
+const collectIfMatches = (collect, element, value) => {
+  const { selector, meta } = value;
+
+  if (matches(selector, element)) {
+    collect(meta);
```

Comment:
What is meta in this case?

Reply:
The reason I ask is that I don't think meta will be captured in the event command options.

Reply:
@liljenback true, it won't work today, but once @Aaronius adds his patch related to `sendEvent` I'll replace this with appropriate invocation.

Today even without click tracking, personalization functionality is half-broken because I can't properly send a notification back to the server since the `event` command doesn't allow `meta` manipulation.

Reply:
@liljenback one more thing, by design Alloy shouldn't really care or know what is in `meta`. The assumption is that `meta` is echoed back to the server without any client side understanding or manipulation.